app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    items_dic = {}
```

Comment:
Rename this variable to `cached_items`, do not add the underlying type without the need.